### DISCUSSION TITLE
How to access a ListView item by index and change a property?

### DISCUSSION THREAD

**Discussion by dvd1221** [flet-dev/flet#1230] [Q&A]
### Question

![image](https://user-images.githubusercontent.com/126869058/228521973-776d5b14-6366-47fa-a53b-99262a6ea385.png)
ListView consists of a set of ElevatedButtons. It is necessary that after selecting the button the previous one goes out. Please help =)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by gbrandt** [comment]
The controls property has a list of controls that the list view displays, you'll have to manually keep track of the currently selected control and then unselect it when a new control is selected.  Listview does not support what you want directly

**Comment by ndonkoHenri** [comment]
Hey!
This could help: https://github.com/flet-dev/flet/discussions/949

If not, post some sample code here so we could help you better.
